cls/SourceControl/Git/Utils.cls

```diff
@@ -183,6 +183,13 @@ ClassMethod IsImportAfter(menuItemName As %String) As %Boolean [ CodeMode = expr
 $Find(..#ImportAfterGitMenuItems, ","_menuItemName_",") > 0
 }
 
+ClassMethod MigrateInstanceSettings()
+{
+    if '$data(@..%SYSNamespaceStorage()@("%gitBinPath")) {
```

Comment:
If there is no gitBinPath configured on the system, this will run every time git-source-control is installed, which could lead to weird behavior. I think instead we should check if any subscripts are defined in the old global (not just "%gitBinPath"). If yes, then we merge the old global to the new and kill the old global.